src/game-engine/components/GamePage/GamePage.js

```diff
@@ -4,41 +4,57 @@ import { Engine } from '../../engine'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import AskCard from '../../../literature/components/AskCard/AskCard'
+import WaitingRoom from '../WaitingRoom/WaitingRoom'
+import { gameActions } from '../../state/actions'
 
 const GamePage = (props) => {
 	let { deal, fold, game } = Engine({})
+	const startGame = (playerId) => {
+		props.startGame(props.gameData.code, playerId)
+	}
 	return (
 		<>
-			<h1>GameCode:{props.gameData.code}</h1>
-			<button
-				style={{
-					position: 'absolute',
-					left: '50vw',
-					top: '50vh'
-				}}
-				onClick={fold}
-			>
-				Fold all
-			</button>
-			<button
-				style={{
-					position: 'absolute',
-					left: '50vw',
-					top: '60vh'
-				}}
-				onClick={() => deal(4)}
-			>
-				Deal
-			</button>
-			{game}
-			{props.cardSelected !== undefined ? <AskCard /> : null}
+			{props.gameData.isActive != true ? (
```

Comment:
Move displaying game code into the waiting room, add it to title there and also call game actions directly from within the waiting room.